models/pathology_tumor_detection/eval/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.npy
+*.txt
```

Comment:
Hi @drbeh , could this `eval` folder be removed? I think it is produced after running the code.

Reply:
Hi @yiheng-wang-nv, actually it is not being produced and causing error at the middle of inference when it want to write down the first probability map result.

Reply:
I see. I just checked the source code of `ProbMapProducer`, it would be nice to enhance it in the next version on making the directory first (like the `SaveImaged` transform does, https://github.com/Project-MONAI/MONAI/blob/a8f67e4278d46b49a87488a41ba38431883e8907/monai/data/folder_layout.py#L95)

What do you think? cc @Nic-Ma @wyli

Reply:
Hi @drbeh ,

I would suggest to remove the `eval` folder in the bundle to be consistent with others, if users face this issue when executing the inference, just let them create the target folder.

Thanks.

Reply:
@yiheng-wang-nv I removed eval folder.